src/extension.ts

```diff
@@ -16,6 +16,13 @@ export function activate(context: vscode.ExtensionContext) {
       jestRunner.runCurrentTest();
     }
   });
+  const runJestAndUpdateSnapshots = vscode.commands.registerCommand('extension.runJestAndUpdateSnapshots', async (argument: object | string) => {
+    if (typeof argument === 'string') {
```

Comment:
why is there this argument parameter? whats its purpose

Reply:
would perfer if we do it like we do it in runJestFileWithCoverage:
https://github.com/firsttris/vscode-jest-runner/blob/65ec6c3b45a93031d8e4ca55944ccb787d3eb9ec/src/extension.ts#L29

Reply:
> why is there this argument parameter? whats its purpose

Removing now.

> would perfer if we do it like we do it in runJestFileWithCoverage:

Fixing this.